app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -132,6 +134,19 @@ class HomePresenter(
         }
     }
 
+    override fun onSendClicked() {
+        if (newSendEnabledFeatureToggle.isFeatureEnabled) {
+            if (state.tokens.isEmpty()) {
+                view?.showSendNoTokens(fallbackUsdcTokenForBuy ?: return)
```

Comment:
In this case if it's null the user will be stuck on the screen? 
If user clicks the app will do anything?